NLog.Windows.Forms/RichTextBoxTarget.cs

```diff
@@ -209,34 +212,59 @@ public RichTextBoxTarget()
         protected override void InitializeTarget()
         {
+
             if (FormName == null)
             {
+                InternalLogger.Info("FormName not set, creating a new form");
                 FormName = "NLogForm" + Guid.NewGuid().ToString("N");
             }
-
-            var openFormByName = Application.OpenForms[FormName];
-            if (openFormByName != null)
+            else
             {
-                TargetForm = openFormByName;
-                if (string.IsNullOrEmpty(ControlName))
+
+                var openFormByName = Application.OpenForms[FormName];
+                if (openFormByName != null)
                 {
-                    throw new NLogConfigurationException("Rich text box control name must be specified for " + GetType().Name + ".");
-                }
+                    bool error = false;
+                    TargetForm = openFormByName;
+                    if (string.IsNullOrEmpty(ControlName))
+                    {
+                        error = true;
+                        var message = "Rich text box control name must be specified for " + GetType().Name + ".";
+                        if (LogManager.ThrowExceptions)
+                        {
+                            throw new NLogConfigurationException(message);
+                        }
+
+                        InternalLogger.Error(message);
+                    }
+
+                    CreatedForm = false;
+                    TargetRichTextBox = FormHelper.FindControl<RichTextBox>(ControlName, TargetForm);
 
-                CreatedForm = false;
-                TargetRichTextBox = FormHelper.FindControl<RichTextBox>(ControlName, TargetForm);
 
-                if (TargetRichTextBox == null)
+                    if (TargetRichTextBox == null)
+                    {
+                        error = true;
+                        var message = "Rich text box control '" + ControlName + "' cannot be found on form '" + FormName + "'.";
+                        if (LogManager.ThrowExceptions)
+                        {
+                            throw new NLogConfigurationException(message);
+                        }
+                        InternalLogger.Error(message);
+                    }
+                    if (!error)
+                        return;
+                }
+                else
                 {
-                    throw new NLogConfigurationException("Rich text box control '" + ControlName + "' cannot be found on form '" + FormName + "'.");
+                    InternalLogger.Info("Form {0} not found, creating a new one", FormName);
                 }
             }
-            else
-            {
-                TargetForm = FormHelper.CreateForm(FormName, Width, Height, true, ShowMinimized, ToolWindow);
-                TargetRichTextBox = FormHelper.CreateRichTextBox(ControlName, TargetForm);
-                CreatedForm = true;
-            }
+            //not done yet
```

Comment:
The comment is indeed confusion. I will fix that.

it should be:
// no form found, we create a new one.